capellambse_context_diagrams/collectors/default.py

```diff
@@ -58,10 +67,10 @@ def collector(
             makers.PORT_PADDING
             + (makers.PORT_SIZE + makers.PORT_PADDING) * len(local_ports),
         )
-        if box := global_boxes.get(i.uuid):
+        if box := global_boxes.get(i.uuid):  # type: ignore[assignment]
```

Comment:
box will never be `None`. The if/else condition makes sure. Mypy not smart enough to figure it out. :(